### ISSUE TITLE
Preprocessing before feeding wav2vec2.0

### ISSUE THREAD

**Issue by revsic** [dhchoi99/NANSY#16]
First, I appreciate your opensource contributions.

[`huggingface:Wav2Vec2ForPretraining`](https://huggingface.co/docs/transformers/model_doc/wav2vec2#transformers.Wav2Vec2ForPreTraining.forward.example) and [`huggingface:Wav2Vec2Model`](https://huggingface.co/docs/transformers/model_doc/wav2vec2#transformers.Wav2Vec2Model.forward.example) use Wav2Vec2Processor or Wav2Vec2FeatureExtractore as their preprocessor. In [git:huggingfac/transformers-Wav2VecFeatureExtractor#L222](https://github.com/huggingface/transformers/blob/v4.12.4/src/transformers/models/wav2vec2/feature_extraction_wav2vec2.py#L222), preprocessor normalize the inputs with their mean and variance if `do_normalize` is True. Also `facebook/wav2vec2-large-xlsr-53` enables it too, `do_normalize`.

```python
>>> ext = Wav2Vec2FeatureExtractor.from_pretrained('facebook/wav2vec2-large-xlsr-53')
>>> ext
Wav2Vec2FeatureExtractor {
  "do_normalize": true,
  "feature_extractor_type": "Wav2Vec2FeatureExtractor",
  "feature_size": 1,
  "padding_side": "right",
  "padding_value": 0,
  "return_attention_mask": true,
  "sampling_rate": 16000
}
```

I trace your codes but I cannot find any audio normalizing routines before feed it to wav2vec2.0 (before [linguistic](https://github.com/dhchoi99/NANSY/blob/master/models/analysis.py#L36), [custom dataset](https://github.com/dhchoi99/NANSY/blob/master/datasets/custom.py#L270) or [inference](https://github.com/dhchoi99/NANSY/blob/master/inference.py#L156)). Is wav2vec2 working without the audio preprocessing ?
